src/backends/onnx/autofill.cc

```diff
@@ -311,6 +312,13 @@ AutoFillOnnx::Create(
 
   RETURN_IF_ORT_ERROR(ort_api->CreateSessionOptions(&session_options));
 
+  // Does nothing if op_libraries is empty
+  for (const auto& lib_filename : op_libraries) {
+    void* library_handle = nullptr;  // leak this, no harm.
```

Comment:
Isn't there a way to unload or close a handle?

Reply:
No such function exists. The use of the handle has not been described. From what I understand it is only exposed to keep the ownership with the user.

Reply:
It would be much more helpful if your comment said something about the use of the handle not being defined... saying it is leaking when we don't really know what is happening is confusing. It would also be good to check with ONNXRT to understand what the use of the handle is for and if any cleanup is needed to unload the libraries when the model is unloaded.